odabot/cli.py

```diff
@@ -129,16 +134,19 @@ def cli(obj, debug, settings):
     )
 
     obj['debug'] = debug
-    
-    logger.info("components: %s", obj['settings'].components)
+    try:
+        logger.info("components: %s", obj['settings'].components)
+    except AttributeError:
+        pass
```

Comment:
could you add some log here?

Reply:
Better to move it to the place where it shouldn't fail